pkg/infinispan/cluster.go

```diff
@@ -314,6 +365,40 @@ func (c Cluster) SetLogger(podName, loggerName, loggerLevel string) error {
 	return nil
 }
 
```

Comment:
It's not clear for me, probably i missed something. Is the second site should be updated already or not?

Reply:
The second site will push it's state if on restart it detects that it's SFS state has been corrupted and recovered. The operator is only concerned with pushing the state of the site it's reconciling.